packages/firebase/index.d.ts

```diff
@@ -5083,6 +5083,7 @@ declare namespace firebase.analytics {
     id?: string;
     name?: string;
   }
+  function isSupported(): boolean;
```

Comment:
Will need a comment here to document what this does. Comments in this file are the source of our official documentation, see other methods/properties for format.